src/vp8_probs.h

```diff
@@ -336,4 +336,240 @@ static const int vp8_bits_per_mb[2][128] =
     }
 };
 
+static const unsigned short vp8_prob_cost[256] =
```

Comment:
There is no problem to included by multi-files because they are static variables.